packages/skeleton-common/src/classes/steps.ts

```diff
@@ -0,0 +1,19 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesSteps = defineSkeletonClasses({
+	root: '',
+	list: 'flex items-center gap-2',
+	item: 'flex items-center gap-2 grow',
+	trigger: 'flex items-center gap-1',
+	indicator: [
```

Comment:
Try these styles to help visualize progression:

```ts
indicator: [
	'size-8 rounded-full preset-outlined-surface-200-800 content-center',
	'data-complete:preset-filled-primary-500',
	'data-current:preset-outlined-primary-500',
],
```

* Outlined Surface 200/800 -> step is not yet reached
* Outlined Primary 500 -> step is active but incomplete
* Filled Primary 500 -> step is complete 

If possible make the `separator` highlight with the primary tone as you progress "through" it. This essentially mimics a timeline or linear progress bar.

Reply:
adding on to this, the `preset-` classes even cause some trouble for ourselves here. For example, the classes you suggested end up with a gray border around the completed indicators, since `data-current:preset-filled...` doesn't override the border from the base `preset-outlined...`. For now, I've changed it to:
```ts
	indicator: [
		'size-8 rounded-full content-center',
		'not-data-complete:not-data-current:preset-outlined-surface-200-800',
		'data-complete:preset-filled-primary-500',
		'data-current:preset-outlined-primary-500',
	],
```
to capture the neither-complete-nor-current state explicitly, so none of the presets are applied at the same time.